fhir-server-test/src/test/resources/nats/docker-compose.yml

```diff
@@ -0,0 +1,50 @@
+version: "3.7"
+networks:
+  main:
+    driver: bridge
+    ipam:
+      config:
+        - subnet: 172.19.0.0/24
+      driver: default
+services:
+    nats-node1:
+        image: nats-streaming:0.17.0
+        container_name: nats-node1
+        volumes:
+            - .:/nats
+        command: --config /nats/config-1.conf
+        networks:
+            - main
+        ports:
+            - 4222:4222
+            - 6222:6222
+    nats-node2:
+        image: nats-streaming:0.17.0
+        container_name: nats-node2
+        volumes:
+            - .:/nats
+        command: --config /nats/config-2.conf
+        networks:
+            - main
+        ports:
+            - 4223:4222
+            - 6223:6222
```

Comment:
```suggestion
            - 4223:4223
```

Reply:
I think its actually listening on 4223 and not 4222

Reply:
The external port is 4223, but the internal port is 4222, so 4223:4222 is correct.

Reply:
Are you sure?  I was basing my assumption on the output from the node which says:
```nats-node2     | [1] 2020/04/24 21:31:08.402708 [INF] Listening for client connections on 0.0.0.0:4223
nats-node2     | [1] 2020/04/24 21:31:08.402753 [INF] Server id is NC5Y56VKV3ECEDMNHBM7LALWBSTWQ2ZSBSPVOMFXJJPZGXRCOKTS5SAO
nats-node2     | [1] 2020/04/24 21:31:08.402770 [INF] Server is ready```